### PR TITLE
[CoreBundle] Administration SSL flag.

### PR DESCRIPTION
**Pull request by ngodfraind**
SSL flag for https detection in case of proxies. Default is false.

### CODE REVIEW

File: main/core/Resources/views/layout.html.twig [hunk 1]
```diff
@@ -25,7 +25,7 @@
 {% block layout %}
     <div id="sf-environement" data-env="{{ app.environment }}" class="hide"></div>
     <div id="homePath" class="hide">{{ path("claro_index") }}</div>
-    {% if app.request.isSecure() %}
+    {% if config.getParameter('ssl_enabled') %}
```
Comment (stefk) [inline main/core/Resources/views/layout.html.twig]: What happens if ssl is used but the parameter isn't set? Shouldn't it be `app.request.isSecure() or config.getParameter('ssl_enabled')` instead?

Comment (ngodfraind) [inline main/core/Resources/views/layout.html.twig]: It's would be better. I need to change 'ssl_enabled' to null by default then.

Comment (stefk) [inline main/core/Resources/views/layout.html.twig]: Why? It should work with a boolean too.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ptsavdar** [comment]
LGTM

**Pull request merged**